contracts/AdjudicationFramework/Push/Feeds.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+/* solhint-disable quotes */
+/* solhint-disable not-rely-on-time */
+
+import {BalanceHolder} from "./../../lib/reality-eth/BalanceHolder.sol";
+
+import {IRealityETH} from "./../../lib/reality-eth/interfaces/IRealityETH.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {MinimalAdjudicationFramework} from "./../MinimalAdjudicationFramework.sol";
+
+/*
+This contract is an example implementation of price feeds using the backstop's arbitration framework.
+*/
+
+contract Feeds is MinimalAdjudicationFramework {
+    uint256 public constant INPUT_SIZE = 5;
```

Comment:
Could do with some comments explaining what this does, eg how the rotating slots work